core/net/rpl/rpl-dag.c

```diff
@@ -1100,6 +1107,19 @@ rpl_recalculate_ranks(void)
     }
     p = nbr_table_next(rpl_parents, p);
   }
+  if (!updated) {
```

Comment:
I don't understand what this does. When no parent needed update, we update the preferred parent of every instance and dag? Clarification needed here